PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -30,6 +30,12 @@ def miniAOD_customizeCommon(process):
     process.patMuons.computeSoftMuonMVA = True
 
     process.patMuons.addTriggerMatching = True
+    from Configuration.Eras.Modifier_run2_muon_2016_cff import run2_muon_2016
+    from Configuration.Eras.Modifier_run2_muon_2017_cff import run2_muon_2017
+    from Configuration.Eras.Modifier_run2_muon_2018_cff import run2_muon_2018
+    run2_muon_2018.toModify( process.patMuons, effectiveAreaVec = cms.vdouble(0.0566, 0.0562, 0.0363, 0.0119, 0.0064))
+    run2_muon_2017.toModify( process.patMuons, effectiveAreaVec = cms.vdouble(0.0566, 0.0562, 0.0363, 0.0119, 0.0064))
+    run2_muon_2016.toModify( process.patMuons, effectiveAreaVec = cms.vdouble(0.0735,0.0619,0.0465,0.0433,0.0577))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

so ```run2_muon_2016.toModify( process.patMuons, effectiveAreaVec = [0.0735,0.0619,0.0465,0.0433,0.0577]```.
Perhaps also keeping 16 -> 17 -> 18 order is better